Sources/XcbeautifyLib/Parser.swift

```diff
@@ -1,156 +1,226 @@
+public enum OutputType {
+    case UNDEFINED
+    case TASK
+    case WARN_ERR
```

Comment:
Could you split this into two `warning` and `error` cases? When adding the `quiet` mode, I was also thinking of adding a new `quieter` mode some time in the future where `xcbeautify` would only prints tasks that have errors. That would make the upcoming change a bit simpler.

Reply:
This is a great idea. Maybe we could figure out levels of verbosity some day.